plugins/omgssapi/omgssapi.c

```diff
@@ -57,7 +57,6 @@
 
 MODULE_TYPE_OUTPUT
 MODULE_TYPE_NOKEEP
-MODULE_CNFNAME("omgssapi")
```

Comment:
Why does imgssapi have ```MODULE_CNFNAME("imgssapi")```?

Reply:
nvm, I see that this was removed as well in an unrelated commit 8d793eafdde0b74d0b4424f0d194b5dc6801a5d1

Reply:
iIRC I was working on removing compile errors...